PRINCIPLES.md

```diff
@@ -0,0 +1,65 @@
+# GitOps Principles v0.1.0
+
+## Summary
+
+GitOps is a set of principles for operating and managing software systems.
+
+When using GitOps, the _Desired State_ of a system or subsystem is defined declaratively as versioned, immutable data, and the running system's configuration is continuously derived from this data.
+
+These principles were derived from modern software operations but are rooted in pre-existing and widely adopted best practices.
+
+## Principles
+
+1. **The principle of declarative desired state**
+
+    A system managed by GitOps must have its _Desired State_ expressed declaritively as data in a format writable and readable by both humans and machines.
+
+2. **The principle of immutable desired state versions**
+
+    _Desired State_ is stored in a way that supports versioning, immutability of versions, and retains a complete version history.
+
+3. **The principle of continuous state reconciliation**
+
+    Software agents continuously, and automatically, compare a system's _Actual State_ to its _Desired State_.
+    If the actual and desired states differ for any reason, automated actions to reconcile them are initiated.
+
+4. **The principle of operations through declaration**
+
+    The only mechanism through which the system is intentionally operated on is through these principles.
+
+## Notes
+
+### Principle 3 Notes
+
+- These differences could be due to the actual state drifting from the desired state, or the desired state changing intentionally.
+- The source of drift doesn't matter. Contrary to CIops, _any_ drift will trigger a reconciliation
+
+### Principle 4 Notes
+
+- We talk here about "regular operations". In an emergency also other mode of operation, e.g. manual intervention, should be considered - followed by a reconsiliation of the "tainted" system with the declared state. → resolve the conflict between "GitOps principle" and "I need to deal with problems that GitOps doesn't cover"
```

Comment:
```suggestion
- We talk here about "regular operations." In an emergency, other modes of operations, e.g. manual intervention, should be considered - followed by a reconciliation of the "tainted" system with the declared state. → resolve the conflict between "GitOps principle" and "I need to deal with problems that GitOps doesn't cover"
```
docs: fix misspellings and grammar
"regular operations". -> "regular operations."
emergency also other mode of operation -> emergency, other modes of operations
reconsiliation -> reconciliation